<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmMain</class>
 <widget class="QWidget" name="FrmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>631</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="searchLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>영화검색</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>511</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>검색어</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtSearch"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearch">
          <property name="text">
           <string>검색</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>99</y>
     <width>631</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="resultLayout">
    <item>
     <widget class="QTableWidget" name="tblResult">
      <property name="font">
       <font>
        <family>새굴림</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
